ussp-api/utm-adapter.yaml

```diff
@@ -158,3 +274,425 @@ components:
+          description: >
+            The state of the operation
+            * `0` - Processing
+            * `1` - Accepted
+            * `2` - Activated
+            * `3` - Nonconforming
+            * `4` - Contingent
+            * `5` - Ended
+
+
+    BadRequest400ErrorResponse:
+      description: The data provided in the request is not valid
+      $ref: '#/components/schemas/Error'
+
+    Unauthorized401ErrorResponse:
+      description: The request requires an user authentication
+
+      $ref: '#/components/schemas/Error'
+
+    Forbidden403ErrorResponse:
+      description: the access is not allowed
+
+      $ref: '#/components/schemas/Error'
+      
+    Standard500ErrorResponse:
+      description: An unexpected error occured.
+      $ref: '#/components/schemas/Error'
+    
+    Error:
+      required:
+        - code
+        - message
+      properties:
+        code:
+          type: string
+        message:
+          type: string
+          
+    FlightDeclarationList:
+      type: array
+      items:
+        $ref: '#/components/schemas/FlightDeclarationDetail'
+
+        
+    FlightDeclarationDetail:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        approved_by:
+          description: Email address of person approving this declaration
+          type: string
+          format: email
+        submitted_by:
+          description: Email address of person submitting this declaration
+          type: string
+          format: email  
+        is_approved:
+          description: Is the operation approved for flight?
+          type: string
+          format: boolean          
+          example: 1
+        start_datetime:
+          description: The start for the query
+          type: string
+          format: date-time
+        end_datetime:
+          description: The enddate for the query
+          type: string
+          format: date-time
+        flight_declaration_geojson:
+          description: The operational intent for this flight declaration as a GeoJSON FeatureCollection
+          type: object
+          properties: 
+            type:
+              type: string
+              example: "FeatureCollection"
+            features:
+              type: array
+              items:
+                $ref: '#/components/schemas/FlightDeclarationFeatureGeoJSON'
+        flight_declaration_raw_geojson:
+          description: The raw / original Flight Plan as GeoJSON
+          type: object
+          properties: 
+            type:
+              type: string
+              example: "FeatureCollection"
+            features:
+              type: array
+              items:
+                $ref: '#/components/schemas/FlightDeclarationFeatureGeoJSON'
+        operational_intent:
+          description: The operational intent for this flight declaration, a raw flight plan is converted to an operational intent which can be in turn submitted to a DSS
```

Comment:
Not sure if its worth mentioning the LF interUSS DSS project  https://github.com/interuss/dss reference somewhere for the context

Reply:
Good point, I have added the reference so that it is clear, the way it is structured, there is a possibility to have other DSS implementations as well that adhere to the ASTM standard.